.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@abhishekkumarstackmybiz files under `.idea` directory should not be checked in as part of the PR. You should include the `.idea` in `.gitignore` file instead.